app/services/manifest_builder/collection_manifest_service_locator.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class ManifestBuilder
+  # Collections have fewer builders (notably the thumbnail builder) to improve
+  # performance.
+  class CollectionManifestServiceLocator < ManifestBuilder::ManifestServiceLocator
+    class << self
+      ##
+      # Builder for a manifest which is a sub-item in a collection.
```

Comment:
Might be useful to explain in a comment that the `thumbnail_builder` is not added / thumbnail urls aren't generated in collection manifests. It's a subtle difference in the code that's easy to miss.

Reply:
Added a comment. Take a look?